File: OLSource1/v1/VS_officedev/codesnippet/CSharp/accessing-a-form-region-at-run-time_2.cs
        private void Access_Form_Regions_ByExplorer()
        {
            WindowFormRegionCollection formRegions =
                Globals.FormRegions
                    [Globals.ThisAddIn.Application.ActiveExplorer()];
            formRegions.FormRegion1.textBox1.Text = "Hello World";
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/accessing-a-form-region-at-run-time_2.vb
    Private Sub Access_Form_Regions_ByExplorer()
        Dim formRegions As WindowFormRegionCollection = Globals.FormRegions _
            (Globals.ThisAddIn.Application.ActiveExplorer())
        formRegions.FormRegion1.textBox1.Text = "Hello World"
    End Sub